components/director/internal/domain/scenarioassignment/fixtures_test.go

```diff
@@ -19,10 +20,13 @@ const (
 )
 
 func fixModel() model.AutomaticScenarioAssignment {
-	return model.AutomaticScenarioAssignment{
+	return fixModelWithScenarioName(scenarioName)
+}
+
+func fixGQLAutomaticScenarioAssignment() graphql.AutomaticScenarioAssignment {
```

Comment:
If my PR comes first, then you can switch to `fixGQL` function; in the other case I will cleanup the fixtures here